### ISSUE TITLE
.tmc-autocomplete.sh should store opts on the disk instead of running tmc

### ISSUE THREAD

**Issue by jclc** [testmycode/tmc-cli#389]
`source $HOME/.tmc-autocomplete.sh` makes starting a new shell take up several seconds. Running the JVM every time you start a shell is not a good idea at all.

**Comment by jclc** [comment]
Also, if tmc is not found, the script hangs and you won't be able to run the shell at all.

**Comment by Salmela** [comment]
The problem most likely is that the tmc tries to auto update. It was fixed in commit 214f880 .

**Comment by Salmela** [comment]
If the tmc doesn't exist. The bash script prints the following on my machine.
`bash: ~/tmc-cli/tmc: No such file or directory`

I think we should hide any error messages from the auto-completion code.
The shell could die due to the strict mode set in your rc file. A workaround for this should be added into the launch script.

**Comment by jclc** [comment]
Yes, that seems to have been fixed in the master branch - I was using the latest release. Might be a good idea to draft a new one.

That being said, even with the latest version tmc still adds a short but noticeable delay when starting a new shell. Probably no big deal for anyone with a sufficiently powerful machine, but I think it would still be best if .tmc-autocomplete didn't run any Java code. Users with toasters may end up having to wait a few seconds. I keep the source line in my .bashrc commented when I'm not actively using it to speed it up.

**Comment by Salmela** [comment]
Yeah It's probably pretty easy to just dump the new commands into a file in the auto-update code. I will look into it.
